lib/ansible/modules/cloud/vmware/vmware_guest_disk.py

```diff
@@ -296,6 +354,25 @@ def reconfigure_vm(self, config_spec, device_type):
 
         return changed, results
 
+    def get_ioandshares_diskconfig(self, disk_spec, disk):
+        if 'iolimit' in disk:
+            ioDiskSpec = vim.StorageResourceManager.IOAllocationInfo()
```

Comment:
```suggestion
            io_disk_spec = vim.StorageResourceManager.IOAllocationInfo()
```

This will match existing  variable scheme.